ludwig/schema/features/preprocessing/image.py

```diff
@@ -109,12 +109,12 @@ class ImagePreprocessingConfig(BasePreprocessingConfig):
         parameter_metadata=FEATURE_METADATA[IMAGE][PREPROCESSING]["infer_image_sample_size"],
     )
 
-    scaling: str = schema_utils.StringOptions(
-        ["pixel_normalization", "pixel_standardization"],
-        default="pixel_normalization",
-        allow_none=False,
-        description="The scaling strategy for pixel values in the image.",
-        parameter_metadata=FEATURE_METADATA[IMAGE][PREPROCESSING]["scaling"],
+    standardize_image: str = schema_utils.StringOptions(
```

Comment:
Since `allow_none` is `True`, set the annotation to `Optional[str]` or `Union[str, None]`. (Should not make a functional difference but good for consistency.)

Reply:
done